### PR TITLE
RSS feed titles and descriptions are now not terrible.

### PR DESCRIPTION
**Pull request by ajrbyers**

### CODE REVIEW

File: src/rss/views.py [hunk 1]
```diff
@@ -51,13 +54,17 @@ def item_link(self, item):
...
 
     def get_object(self, request, *args, **kwargs):
         return request.journal
 
+    def title(self, obj):
+        return "{} Article Feed".format(obj.name)
+
+    def description(self, obj):
+        return "A feed of the 10 latest articles from {}".format(obj.name)
+
```
Comment (mauromsl) [inline src/rss/views.py:67]: can `obj` be `None` ? It seems that way from `self.get_object`, in which case it would raise an `AttributeError` here

**Comment by mauromsl** [comment]
jenkins: test this please

**Comment by ajrbyers** [comment]
Looks like this one is good to go!

**Comment by mauromsl** [comment]
> Looks like this one is good to go!

Waiting on Jenkins at the moment, there were a few tests still failing

**Comment by ajrbyers** [comment]
> > Looks like this one is good to go!
> 
> Waiting on Jenkins at the moment, there were a few tests still failing

Ah, it showed passed to me!

**Comment by mauromsl** [comment]
> > > Looks like this one is good to go!
> > 
> > 
> > Waiting on Jenkins at the moment, there were a few tests still failing
> 
> Ah, it showed passed to me!

Shows as pending to me, but you are right, build has passed! 💚

**Pull request merged**